src/metabase/models/dashboard_card.clj

```diff
@@ -170,7 +170,7 @@
   (into {}
         (filter (fn [[k v]]
                   (not= v (get old k)))
-        new)))
+         new)))
```

Comment:
```suggestion
        new)))
```

Reply:
`new` is the arg for `filter` so it should be indent even further https://github.com/metabase/metabase/pull/31060/commits/0ce5b08b1f251837d6e5f2f08290137ca7ffb378

Reply:
Ugh, paren counting is hard. I assumed you were using filter as a transducer with two args